cover/cover.go

```diff
@@ -58,7 +58,7 @@ func (c *Cover) Report(modules map[string]*ast.Module) (report Report) {
 			return false
 		})
 		ast.WalkExprs(module, func(x *ast.Expr) bool {
-			if hasFileLocation(x.Location) {
+			if includeExprInCoverage(x) && hasFileLocation(x.Location) {
```

Comment:
I'd move the `hasFileLocation` check into this new helper function.

Reply:
Done